README.md

```diff
@@ -49,28 +53,35 @@ $ prism mock examples/petstore.oas3.json
 Then in another tab, you can hit the HTTP server with your favorite HTTP client (like [HTTPie]):
```

Comment:
Is that suppose to be a link? Or do we still need the HTTPie mention?

Reply:
Good catch! Fixed!